### ISSUE TITLE
AUC metric

### ISSUE THREAD

**Issue by zeyevv** [jarrycyx/UNN#12]
I have a question about model training. When I tested CUTS using the DREAm dataset, I found that AUC always rises first and then falls. Is this phenomenon normal? Are there any other metrics to measure the effectiveness of causal inference？

**Comment by jarrycyx** [comment]
Yes, we observed the same phenomenon. We think this may be because of the small sample size of DREAM data. So early stopping may be helpful.
For "the effectiveness of causal inference", do you mean computing ATE, ITE, etc?
